ansible/group_vars/all

```diff
@@ -114,6 +114,17 @@ kafka:
   ras:
     port: 9093
 
+# Kafka retention for topics : completed, health and invoker
+kafka_topic_completed_logRetentionBytes: 1073741824
+kafka_topic_completed_logRetentionMS: 3600000
+kafka_topic_completed_logSegmentBytes: 536870912
+kafka_topic_health_logRetentionBytes: 1073741824
+kafka_topic_health_logRetentionMS: 3600000
+kafka_topic_health_logSegmentBytes: 536870912
+kafka_topic_invoker_logRetentionBytes: 1073741824
+kafka_topic_invoker_logRetentionMS: 172800000
+kafka_topic_invoker_logSegmentBytes: 536870912
```

Comment:
Why don't we put these into a dictionary rather than a flat map of properties?

```
kafka:
  topics:
    completed:
        segmentBytes: 536870912
        retentionBytes: 1073741824
        retentionMs: 3600000
...
```

That's imho much more readable. If we want those to be overridable, we can use a flat property like a above as the default:

```
kafka:
  topics:
    completed:
        segmentBytes: "{{ kafka_topics_completed_retentionBytes | default(536870912) }}"
        retentionBytes: 1073741824
        retentionMs: 3600000
...
```